content/rs/administering/troubleshooting/database-recovery.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: Recovering a Failed Database
+description: 
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When a cluster fails or a database is corrupted,
+you must recover the databases with their previous configuration and data.
+
+Database recovery requires the database persistence files that are created by the cluster.
+These files are stored in the [persistence storage location]
+({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}}).
+
+The database recovery process includes:
+
+1. If the cluster failed, [recover the cluster]({{< relref "/rs/administering/troubleshooting/cluster-recovery.md" >}}).
+1. Identify recoverable databases.
+1. Restore the database configurations.
+1. Verify that the databases are active.
+
+## Prerequisites
+
+- Before you start database recovery, make sure that the cluster that hosts the database is healthy.
+In the case of a cluster failure,
+you must [recover the cluster]({{< relref "/rs/administering/troubleshooting/cluster-recovery.md" >}}) before you recover the databases.
+
+- We recommend that you allocate new persistent storage drives for the new cluster nodes.
+If you decide to use the persistent storage drives of the old cluster nodes,
+make sure that you backup all files on the old persistent storage drives to another location.
+
+## Recovering the Databases
+
+After you prepare the cluster that hosts the database,
+you can run the recovery process from the [rladmin]({{< relref "/rs/references/cli-reference/rladmin.md" >}})
+command-line interface (CLI).
+
+{{% note %}}
+Before you run the recovery process on the machines from the original cluster with Redis processes running on the machine,
+make sure to kill all Redis processes.
+{{% /note %}}
+
```

Comment:
I don't think this is needed for database recovery, only for cluster recovery.

Reply:
Do you want to confirm it or should I remove it?

Reply:
Not sure how to verify it. You can maybe ask someone from support.